generation/check_owlbot_config.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Checks that each module's .OwlbotYaml file is correctly configured according to the template set in `set_owlbot_config.sh`
+
+OWLBOT_FILE=$1
+for F in $(find . -maxdepth 2 -name '.OwlBot.yaml'); do sh generation/set_owlbot_config.sh "$F"; done
```

Comment:
Since this actually modifies the files, and doesn't really check, I think the file name doesn't really make sense. Would it make sense to just add this loop to `set_owlbot_config.sh` and update places where it's used? I think we always want to apply this to all `.OwlBot.yaml` files.

Reply:
Hmm I see what you're saying; I did a little more digging and it looks like `set_owlbot_config.sh` is actually run as part of `merge_repository.sh` (https://github.com/googleapis/google-cloud-java/blob/main/generation/merge_repository.sh#L121) which is run regularly as a workflow: https://github.com/googleapis/google-cloud-java/blob/main/.github/workflows/merge_repository.yaml#L22. 

My sense is that actually would suffice as the check for the Owlbot.yaml files - what do you think?

Reply:
That check doesn't apply to the entire repo and might go away in the future. I think the check your added is valuable beyond that.

Reply:
Thanks for pointers offline - let me know if this update works!

Reply:
I see that you updated `merge_repository.sh`, but `bootstrap.sh` still needs an update.

Reply:
Right, never mind! I think it's still in my local for some reason.